builds/msvc/vs2013/libbitcoin-explorer/libbitcoin-explorer.vcxproj

```diff
@@ -79,6 +79,7 @@
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer.hpp" />
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer\impl\prop_tree.ipp" />
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer\impl\utility.ipp" />
+    <ClInclude Include="..\..\..\..\include\bitcoin\explorer\async_client.hpp" />
```

Comment:
Sorry, the BX generation is manually-specified. I'm used to working with libbitcoin-build, which is similar. I have to rationalize these at some point. You'll need to edit the model generation code for the BX file changes.

You should be working from `libbitcoin-build/master`, which does not have `async_client`.